docs/user_guide/index.rst

```diff
@@ -226,30 +226,32 @@ on a predetermined set while remaining aware of the others.
 
 This audit should be used by experts with a deep understanding of the
 model and the context in which the predictions are used. Even when a
-metric is flagged as failing in the fairness audit, there might be
-context that explains and even predicts the difference. Like many
+metric is flagged as a deviation in the fairness audit, the context 
+might that explains or even predict the difference. Like many
```

Comment:
"might explain or even predict" (remove "that" and final "s" from "explains").

Reply:
FINC